buildenv/jenkins/testJobTemplate

```diff
@@ -225,9 +225,12 @@ ARCH_OS_LIST.each { ARCH_OS ->
 							stringParam('OPENJDK_SHA', "", "OpenJDK SHA from which to run tests")
 							stringParam('TKG_OWNER_BRANCH', "AdoptOpenJDK:master", "TestKitGen git [owner]:[branch]. Default is AdoptOpenJDK:master")
 							stringParam('TKG_SHA', "", "TKG sha")
+              stringParam('ADOPTOPENJDK_SYSTEMTEST_OWNER_BRANCH', "Personal Adoptopenjdk/openjdk-systemtest repository owner:branch", "AdoptOpenJDK:master")
+              stringParam('OPENJ9_SYSTEMTEST_OWNER_BRANCH', "Personal eclipse/openj9-systemtest repository owner:branch", "eclipse:master")
+              stringParam('STF_OWNER_BRANCH', "Personal stf repo owner:branch", "AdoptOpenJDK:master")
```

Comment:
The definition of stringParam is `stringParam(String parameterName, String defaultValue = null, String description = null)` https://jenkinsci.github.io/job-dsl-plugin/#path/job-parameters-stringParam. 

So the second and third parameters are in the wrong order.

Reply:
Another questions is if those parameters need to be set as a variable so they can be passed to auto Gen?  See the variable JDK_REPO as a example.